Source/WebKit/UIProcess/API/glib/WebKitWebContext.cpp

```diff
@@ -2023,7 +2023,7 @@ WebProcessPool& webkitWebContextGetProcessPool(WebKitWebContext* context)
 
 void webkitWebContextCreatePageForWebView(WebKitWebContext* context, WebKitWebView* webView, WebKitUserContentManager* userContentManager, WebKitWebView* relatedView, WebKitWebsitePolicies* defaultWebsitePolicies)
 {
-    auto pageConfiguration = API::PageConfiguration::create();
+    auto pageConfiguration = relatedView ? webkitWebViewGetConfigurationForNextRelatedView(relatedView) : API::PageConfiguration::create();
```

Comment:
Note that user can create a web view with a related view without being in create callback, so we can have a related view but not a page configuration.

Reply:
Yes, this is what Alex was concerned about. Thanks.